lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts

```diff
@@ -993,6 +993,41 @@ describe('StartProcessCloudComponent', () => {
 
             expect(card).toBeTruthy();
         });
+
+        it('start process button should be enabled when isLoading is false', async () => {
+            component.ngOnChanges({ appName: firstChange });
+            component.processDefinitionList = fakeProcessDefinitions;
+            component.processDefinitionName = fakeProcessDefinitions[0].name;
+            component.ngOnInit();
+            component.processForm.controls['processInstanceName'].setValue(fakeProcessDefinitions[0].id);
+            component.appName = 'test app name';
+            component.isLoading = false;
+            fixture.detectChanges();
+            await fixture.whenStable();
+
+            const startButton = fixture.debugElement.query(By.css('#button-start'));
+            expect(startButton).not.toBeNull();
+            expect(component.disableStartButton()).toBeFalse();
+            expect((startButton.nativeElement as HTMLButtonElement).disabled).toBeFalse();
+        });
+
+        it('start process button should be disabled when isLoading is true', async () => {
+            component.ngOnChanges({ appName: firstChange });
+            component.processDefinitionList = fakeProcessDefinitions;
+            component.processDefinitionName = fakeProcessDefinitions[0].name;
+            component.ngOnInit();
```

Comment:
this one is not needed usually

Reply:
As above is needed because in ngOnInit there is a code related to setting process definition name which is used in further evaluation of whether start button should be enabled or not

Reply:
fixed